.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         run: |
           sudo su
           mkdir bin && cd bin
-          cmake .. -G Ninja -DCMAKE_TOOLCHAIN_FILE=../CMakeLists_mingw.txt -DCMAKE_BUILD_TYPE=MinSizeRel -DDISABLE_IPO=on -DFORCE32=on -DBUILD_SHARED_LIBS=ON
+          cmake .. -G Ninja -DCMAKE_TOOLCHAIN_FILE=../CMakeLists_mingw.txt -DCMAKE_BUILD_TYPE=MinSizeRel -DDISABLE_IPO=on -DFORCE32=on -DBUILD_SHARED_LIBS=ON -DENABLE_SCRIPTING=OFF
```

Comment:
It depends what you mean by doesn't support it. This only disables CI building of it, if you were to locally use mingw and you had the duktape package available, it should work.

Reply:
I see. It looks weird to me that this is the only integration not testing scripts. If that seems ok, then we can go with it, otherwise maybe file an issue to follow-up, please?

Reply:
Android also does not have it enabled, so two integrations. I would enable it for mingw but duktape needs adding to the docker image. I can create an issue for doing that. We don't supply the mingw builds from CI.

Reply:
Please create an issue then and we leave it as is for now, thanks :)